src/test/java/org/ays/tests/institution/assignmentmanagementservice/PostAssignmentTest.java

```diff
@@ -48,7 +48,7 @@ public void createAssignmentWithInvalidDescription(String invalidDescription, St
                 .body("subErrors[0].type", equalTo("String"));
     }
 
-    @Test(dataProvider = "invalidCountryCodeData")
+    @Test(dataProvider = "invalidCountryCodeData",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html